caffe2/operators/quantized/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# ---[ CPU files only
+set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS}
```

Comment:
Nit: This can actually be `list(APPEND ...`

Reply:
fixed